frontend/public/components/utils/request-size-input.tsx

```diff
@@ -59,4 +59,5 @@ export type RequestSizeInputProps = {
   defaultRequestSizeUnit: string;
   defaultRequestSizeValue: string;
   describedBy?: string;
+  step?: number;
```

Comment:
Are these changes required, if we are disabling button.
We must avoid making changes to generic components of console

Reply:
@afreen23, the OSD should be increased in factors of 2, so just to make a better user experience when up/down arrow keys are pressed, I have added `step` as an optional parameter. Since it can still have `any` as value, it is not affecting any other component where it is used. @spadgett what are your thoughts about this ?

Reply:
This should be fine since the behavior is unchanged if I don't provide a `step` prop